File: app/src/main/java/net/ivpn/client/vpn/model/WifiItem.java
package net.ivpn.client.vpn.model;

import net.ivpn.client.common.utils.StringUtil;

public class WifiItem {

    private String ssid;
    private String title;
    private NetworkState state;

    public WifiItem(String ssid, NetworkState state) {
        this.state = state;
        this.ssid = ssid;
        title = StringUtil.formatWifiSSID(ssid);
    }

    public String getSsid() {
        return ssid;
    }

    public String getTitle() {
        return title;
    }

    public NetworkState getNetworkState() {
        return state;
    }

    public void setNetworkState(NetworkState state) {
        this.state = state;
    }
}

File: app/src/main/java/net/ivpn/client/vpn/model/WifiItem.kt
package net.ivpn.client.vpn.model

import androidx.databinding.ObservableField
import net.ivpn.client.common.utils.StringUtil

data class WifiItem(val ssid: String, var networkState: ObservableField<NetworkState>) {
    val title: String = StringUtil.formatWifiSSID(ssid)
}